CHANGES.md

```diff
@@ -1,3 +1,32 @@
+## edge-18.12.3
+
+Upgrade notes: The control plane components have been renamed as of the
+edge-18.12.1 release to reduce possible naming collisions. To upgrade an
+older installation, see the [Upgrade Guide](https://linkerd.io/2/upgrade/).
+
+* CLI
+  * Multiple improvements to the `linkerd install` config (thanks @codeman9!)
+    * Use non-default service accounts for grafana and web deployments
+    * Use `emptyDir` volume mount for prometheus and grafana pods
+    * Set security context on control plane components to not run as root
+  * Remove cluster-wide resources from single-namespace installs
+    * Disable service profiles in single-namespace mode
+    * Require that namespace already exist for single-namespace installs
+  * Fix resource requests for proxy-injector container in `--ha` installs
+* Controller
+  * Block controller initialization until caches have synced with kube API
+  * Fix proxy-api handling of named target ports in service configs
+  * Add parameter to stats API to skip retrieving prometheus stats (thanks,
+    @alpeb!)
+* Web UI
+  * Adjust label for unknown routes in route tables, add tooltip
+  * Update Top Routes page to persist form settings in URL
+  * Add button to create new service profiles on Top Routes page
+  * Fix CLI commands displayed when linkerd is running in non-default namespace
+* Proxy
+  * Disable protocol detection for server-speaks-first protocols on default
```

Comment:
I'd rephrase this as something like "Proxies with TLS enabled now honor ports which skip protocol detection"; this change fixes both the default skipped ports _and_ any user-configured ones.

Reply:
(also it's not actually on master yet, due to issues w/ the proxy CI build)

Reply:
we'll wait until this gets in, update the proxy version and then ship the release! (or if this change won't get in before eod Friday, we'll release without it, and remove this bullet point).

Reply:
#1989 updates the pinned proxy version to pick up this change :)